src/date-range-picker/interfaces.ts

```diff
@@ -69,68 +124,18 @@ export interface DateRangePickerProps extends BaseComponentProps, FormFieldValid
    */
   onBlur?: NonCancelableEventHandler<null>;
 
-  /**
-   * Fired whenever a user changes the component's value.
-   * The event `detail` contains the current value of the field.
-   */
-  onChange?: NonCancelableEventHandler<DateRangePickerProps.ChangeDetail>;
-
   /**
    * A function that defines whether a particular range is valid or not.
    *
    * Ensure that your function checks for missing fields in the value.
    */
   isValidRange: DateRangePickerProps.ValidationFunction;
```

Comment:
Actually, why this is not supported for the embedded component?

Reply:
And also startOfWeek

Reply:
For the embedded component, we remove the validation section. `isValidRange` is used to determine the validation result which only gets used in the validation section. 

Do you think this should be added as a base prop? (Though it would not be used?)

Reply:
For start of week, we can actually move the normalize start of week behaviour into the calendar itself (like we have done with date picker). Will create a separate PR for that